spec/fixtures/iso-pubid-directives.txt

```diff
@@ -2,8 +2,11 @@ ISO/IEC DIR 1
 ISO/IEC DIR 1:2022
 ISO/IEC DIR 2
 ISO/IEC DIR 2:2021
+ISO/IEC DIR 2 IEC
+ISO/IEC DIR 2 ISO
```

Comment:
We also have year editions `ISO/IEC DIR 1 ISO:2002`. Add it to the test, please.

Reply:
@andrew2net done